test/unit-test/test_btc_sign.c

```diff
@@ -1452,7 +1482,13 @@ static void _test_antiklepto(void** state)
     valid.antikepto = true;
     _sign(&valid);
 }
-
+static void _test_p2tr_output(void** state)
+{
+    _modification_t invalid = _valid;
+    invalid.p2tr_output = true;
+    invalid.check_sigs = true;
+    _sign(&invalid);
```

Comment:
slightly confused. what does it make initial the `_valid` invalid?

Reply:
fixed in a new commit, renaming also all other such vars.